lnbits/core/models.py

```diff
@@ -213,3 +213,14 @@ class BalanceCheck(BaseModel):
     @classmethod
     def from_row(cls, row: Row):
         return cls(wallet=row["wallet"], service=row["service"], url=row["url"])
+
+
+class TinyURL(BaseModel):
+    id: str
+    url: str
+    endless: bool
+    time: float
```

Comment:
- I cannot find where `endless` and `time` are used. Can these fields be dropped?

Reply:
https://github.com/lnbits/lnbits/pull/1361#issuecomment-1401954793